### PR TITLE
Add eyeglass support.

### PR DESCRIPTION
**Pull request by chriseppstein**
This pull request adds the necessary support and documentation for users of [Eyeglass](http://eyeglass.rocks) as discussed in #774.

### CODE REVIEW

File: eyeglass-exports.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+var bourbon = require(".");
+
+module.exports = function(eyeglass, sass) {
```
Comment (chriseppstein) [inline eyeglass-exports.js]: Yet :stuck_out_tongue:

File: eyeglass-exports.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+var bourbon = require(".");
```
Comment (chriseppstein) [inline eyeglass-exports.js]: it is defined by magic :sparkles:

File: eyeglass-exports.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+module.exports = function(eyeglass, sass) {
+  return {
+    sassDir: bourbon.includePaths[0]
+  }
```
Comment (chriseppstein) [inline eyeglass-exports.js]: fine. fixed.

**Comment by whmii** [comment]
As a note since that url dosent go anywhere https://github.com/sass-eyeglass/eyeglass

**Comment by chriseppstein** [comment]
@whmii fixed the link :blush:

File: eyeglass-exports.js (hunk 2 above)
Comment (tysongach) [inline eyeglass-exports.js]: :smile:

**Comment by tysongach** [comment]
Thanks @chriseppstein!

Should we combine with or remove [`index.js`](https://github.com/thoughtbot/bourbon/blob/master/index.js) now? They seem redundant or too similar and I’m not sure I see the value in keeping both?

I’m gonna toss up one commit, if you don’t mind, to align these new instructions in our README with the existing ones.

**Comment by chriseppstein** [comment]
@tysongach My goal was to avoid breaking anything. I assumed someone was using the existing require functionality, the way I coded this up at least, it's DRY.

**Comment by chriseppstein** [comment]
@tysongach I added you as a collaborator on https://github.com/chriseppstein/bourbon/tree/eyeglass so that you can make the README changes there if you want to.

**Comment by tysongach** [comment]
@chriseppstein Nice, thanks for that. Just tossed up that commit.

This is looking good!

**Comment by chriseppstein** [comment]
:+1: Anything else needed to merge?

**Comment by tysongach** [comment]
Merged! Thanks again @chriseppstein!

**Comment by chriseppstein** [comment]
@tysongach yay! thanks.

**Pull request closed without merging**